### DISCUSSION TITLE
Cannot customize button bg color

### DISCUSSION THREAD

**Discussion by VikaramjitSinghGorraya** [chakra-ui/chakra-ui#4073] [Q&A]
I am trying to customize the bg color of Button component. Here is the code that I have written: 

import { theme as chakraTheme } from '@chakra-ui/react'
import { extendTheme } from "@chakra-ui/react"
import { createBreakpoints } from "@chakra-ui/theme-tools"

const fonts = {
    ...chakraTheme.fonts,
    body: `Raleway`,
    heading: `Farro`
}

const breakpoints = createBreakpoints({
    sm: "40em",
    md: "52em",
    lg: "64em"
})

const overrides = {
    ...chakraTheme,
    fonts,
    breakpoints,
    components: {
        Link: {
            baseStyle: {
                _hover: {
                    textDecoration: "none"
                }
            }
        },
        **_Button: {
            variants: {
                bg: "red",  
            }_**
        }
    },
    fontWeights: {
        normal: 300,
        medium: 600,
        bold: 700
    },
    fontSizes: {
        xs: "12px",
        sm: "14px",
        md: "16px",
        lg: "18px",
        xl: "20px",
        "2xl": "24px",
        "3xl": "28px",
        "4xl": "36px",
        "5xl": "48px",
        "6xl": "64px",
    },
}

const customTheme = extendTheme(overrides)

export default customTheme

**Comment by vcastroi** [comment]
are you trying to setup a new variant, update one of the existing variants  ? 
if changing the solid variant
```
Button: {
variants: {
 solid: (props) => ({
          bg: props.colorMode === "dark" ? "red.300" : "red.500",
        }),
}
```
^ this uses different shades of red for dark and light color mode
